### PR TITLE
Add test for Random Number Generator

### PR DESCRIPTION
**Pull request by ShouvikGhosh2048**
@ayushpatnaikgit 
This PR adds a test for CRRao's set_rng and CRRao_rng.

### CODE REVIEW

File: test/numerical/RandomNumberGenerator.jl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CRRao.set_rng(StableRNG(123))
+@test rand(CRRao.CRRao_rng, Float64, 1) ≈ [0.18102554215580358]
```
Comment (ShouvikGhosh2048) [inline test/numerical/RandomNumberGenerator.jl:2]: @ayushpatnaikgit 
I'm performing an approximate equality check here. Do you want exact equality instead?

Comment (sourish-cmi) [reply test/numerical/RandomNumberGenerator.jl:2]: approximate equality check should be fine

**Comment by ayushpatnaikgit** [comment]
Good to merge.

**Review by sourish-cmi — APPROVED** [review]
All checks have passed. Good to merge.

**Pull request merged**